ingredients.js

```diff
@@ -34,16 +32,84 @@ export const createNodeIngredients = () => {
     ingredientsList.appendTo(content).hide()
 }
 
-const createNewIngredientsItem = getText => {
-    const list = $('<li>', {
-        class: 'ingredients'
-    }).append($('<p>', {
-        class: 'ingredients__list',
-        text: getText,
-        'data-id': uuidv4()
-    }))
+const sendIngredientsToBackEnd = inputText => {
+    const ingredient = fetch('http://localhost:3001/api/ingredient', {
+        method: 'POST',
+        body: JSON.stringify({ ingredientName: inputText }),
+        headers: { "Content-Type": "application/json" }
+    })
+        .then(res => {
+            if (res.ok) {
+                return res.json()
+            }
+            throw error
+        })
+        .catch(error => {
+           alert(error)
```

Comment:
you should also return eg. null, so your ingredient returned from function will be null, you can handle it in the next function